configure.ac

```diff
@@ -414,8 +414,8 @@ if test x$use_pkgconfig = xyes; then
   ])
 fi
 
-if test x$use_extended_rpc_tests != xno; then
-  AC_SUBST(EXTENDED_RPC_TESTS, -extended)
+if test x$use_extended_functional_tests != xno; then
+  AC_SUBST(EXTENDED_FUNCTIONAL_TESTS, -extended)
```

Comment:
Please update Makefile.am to match. I'm pretty sure the cov stuff is busted atm, but at least this part won't be busted too.

Actually, it'd make sense to use this in test/functional/config.ini.in directly if you'd prefer to skip to that.